Mage.Sets/src/mage/sets/kaladesh/RushOfVitality.java

```diff
@@ -0,0 +1,39 @@
+package mage.sets.kaladesh;
+
+import mage.abilities.effects.common.continuous.BoostTargetEffect;
+import mage.abilities.effects.common.continuous.GainAbilityTargetEffect;
+import mage.abilities.keyword.DeathtouchAbility;
+import mage.abilities.keyword.IndestructibleAbility;
+import mage.abilities.keyword.LifelinkAbility;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Rarity;
+import mage.target.common.TargetCreaturePermanent;
+
+import java.util.UUID;
+
+/**
+ * Created by IGOUDT on 16-9-2016.
+ */
+public class RushOfVitality extends CardImpl {
+
+    public RushOfVitality(final UUID ownerId){
+        super(ownerId,99,"Rush of Vitality", Rarity.COMMON, new CardType[]{CardType.INSTANT},"{1B}");
```

Comment:
You don't use the gen-card.pl script?
The card text is missing please add that.